content/en/docs/tasks/administer-cluster/nodelocaldns.md

```diff
@@ -56,13 +56,13 @@ This feature can be enabled using the following steps:
 * Prepare a manifest similar to the sample [`nodelocaldns.yaml`](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml) and save it as `nodelocaldns.yaml.`
 * Substitute the variables in the manifest with the right values: 
 
-   kubedns=`kubectl get svc kube-dns -n kube-system -o jsonpath={.spec.clusterIP}` 
+     * kubedns=`kubectl get svc kube-dns -n kube-system -o jsonpath={.spec.clusterIP}` 
    
-   domain=`<cluster-domain>` 
+     * domain=`<cluster-domain>` 
    
-   localdns=`<node-local-address>`
+     * localdns=`<node-local-address>`
   
-   `<cluster-domain>` is "cluster.local" by default. `<node-local-address>` is the local listen IP address chosen for NodeLocal DNSCache.
+     `<cluster-domain>` is "cluster.local" by default. `<node-local-address>` is the local listen IP address chosen for NodeLocal DNSCache.
```

Comment:
@prameshj , looking good. Double check that the list items and sub items line up.

Reply:
Thanks Karen! This looks correct when i check the preview in https://github.com/kubernetes/website/blob/67fca6dd44be0eac91b405b11f001436add53809/content/en/docs/tasks/administer-cluster/nodelocaldns.md

Does https://deploy-preview-18716--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/nodelocaldns/ update automatically as well?

Reply:
Looks better, thanks!

Reply:
Thanks Karen! Would you be able to approve this PR?